_sass/_blog.scss

```diff
@@ -0,0 +1,179 @@
+.blog-sidebar {
+  background-color: $navy-drk;
+  border-radius: 2px;
+  height: 100%;
+  margin: {
+    top: 35px;
+    bottom: 48px;
+    right: 60px;
+  }
+}
+
+.recent-post-header {
+  font-size: 20px;
+  padding-left: 25px;
+}
+
+h3.recent-post-header {
+  margin: {
+    top: 10px;
+    bottom: 13px;
+  }
+}
+
+.recent-posts {
+  background-color: lighten($navy-drk, 3%);
+  padding: {
+    top: 5px;
+    bottom: 5px;
+  }
+  h2 {
+    padding-left: 25px;
+    font-size: 10px;
+    a {
+      color: $green;
+    }
+  }
+}
+
+.post,
+.index-list {
+  width: 100%;
+}
+
+.index-list {
+  display: flex;
+  flex-direction: column;
+  h1 {
+    margin-bottom: 0px;
+    margin-top: 22px;
+  }
+  p.meta {
+    margin-top: 0px;
+  }
+  .post {
+    margin-bottom: 30px;
+  }
+}
+
+.blog-posts,
+.post-wrapper {
+  display: flex;
+  flex-direction: row;
+}
+
+.post h1 a {
+  color: white;
+  font-size: 36px;
+  &:hover {
+    color: $green;
+    text-decoration: none;
+    border-bottom: 2px solid $green;
+  }
+}
+
+.post p.meta {
+  color: #8FA5BC;
+}
+
+.post h1,
+.post h2,
+.post h3,
+.post h4 {
+  color: #8FA5BC;
+}
+
+.post-header {
+  margin: 0px 0 30px;
+
+  h1 {
+    margin-bottom: 0px;
+    color: white;
+    font-size: 48px;
+    letter-spacing: -1.75px;
+    line-height: 1;
+    font-weight: 300;
+  }
+
+  .meta {
+    font-size: 15px;
+    color: #818181;
+    margin-top: 5px;
+  }
+}
+
+.post-content {
+  margin: 0 0 30px;
+}
+
+.post-content > * {
+  margin: 20px 0;
+}
+
+.post-content h1,
+.post-content h2,
+.post-content h3,
+.post-content h4,
+.post-content h5,
+.post-content h6 {
```

Comment:
Same with all of these heading rules: please move them into the `.post-content` block as nested blocks (as well as all the other instances below).